src/main/java/frc/team4909/robot/subsystems/elevatorarm/commands/SetAngle.java

```diff
@@ -16,4 +16,5 @@ public SetAngle(int setpoint, ElevatorArmSubsytem elevatorArm) {
     public void initialize() {
         elevatorArm.holdingPosition = setpoint;
     }
+
```

Comment:
same as above with the exit condition– this idea worked well last year but ideally the command should only exit when it's within an acceptable threshold and the action can be considered "complete"